File: 2d a 1d columna/columna.js
let matriz2D = [[1,2,3],[4,5,6],[7,8,9]];
let matriz1D = [];

console.log("Matriz 2D original:");
matriz2D.forEach(fila => console.log(fila));

for (let col = 0; col < matriz2D[0].length; col++) {
    for (let fila = 0; fila < matriz2D.length; fila++) {
        matriz1D.push(matriz2D[fila][col]);
    }
}

console.log("\nMatriz 1D (por columnas):");
console.log(matriz1D);

File: 2d a 1d columna/columna.py
matriz2D = [[1,2,3],[4,5,6],[7,8,9]]
matriz1D = []

print("Matriz 2D original:")
for fila in matriz2D:
    print(fila)

for col in range(len(matriz2D[0])):
    for fila in range(len(matriz2D)):
        matriz1D.append(matriz2D[fila][col])

print("\nMatriz 1D (por columnas):")
print(matriz1D)
